pages/dkp/kommander/2.2/insights/installation-configuration/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle: Insights installation and configuration
+title: Insights installation and configuration
+excerpt: Insights installation and configuration
+menuWeight: 30
+beta: false
+enterprise: false
+techPreview: true
+---
+
+DKP Insights is enabled by default with the installation of the DKP platform. No other actions are required.
```

Comment:
The Technology Preview of DKP insights is included with the installation of the DKP platform. 
I think the below sentence is already addressed in the next section
The Management/frontend of the DKP insights is enabled by default but, the DKP insights engine needs to be manually enabled in all the attached Kubernetes clusters in DKP

Reply:
I think we should use the same terminology we use in other places.   IIUC, Insights is installed by default on the Management cluster; it must be manually enabled on managed clusters.    Obviously, we will need information on how to do that...